.goreleaser.yml

```diff
@@ -138,8 +138,15 @@ nfpms:
           mode: 0750
           owner: observiq-otel-collector
           group: observiq-otel-collector
+      - src: scripts/package/service/systemd/observiq-otel-collector.service
+        dst: /usr/lib/systemd/system/observiq-otel-collector.service
+        type: config
```

Comment:
I'm kind of torn on whether this should be `config` or `config|noreplace`. I think `config|noreplace` is what is expected (the users changes are preferred no matter what, even if we change the service file).

The way it is right now, if we change the service file in our package, it will overwrite the user's changes (for RPM, dpkg will prompt the user to decide either way).

I did some digging, and ElasticSearch marks their service file as `config|noreplace`, so maybe we should too.